File: api/webAPI/WebApplication1/Models/BezbednosniDokumentLog.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication1.Models
{
    public class BezbednosniDokumentLog
    {
        public int idBezbednosnogDokumenta { get; set; }
        public DateTime DatumPromene { get; set; }
        public int idKorisnika { get; set; }
        public string Status { get; set; }
    }
}

File: angular-web2-app/src/app/entities/bezbednosni-dokument-log/bezbednosni-dokument-log.ts
export class BezbednosniDokumentLog {
    idBezbednosnogDokumenta:number;
    Status:string;
    //idKorisnika:number;
    DatumPromene:Date;

    constructor(idBezbednosnogDokumenta:number, Status:string, DatumPromene:Date) {
        this.idBezbednosnogDokumenta = idBezbednosnogDokumenta;
        this.Status = Status;
        this.DatumPromene = DatumPromene;
    }
}
